samples/cellular/nrf_cloud_mqtt_multi_service/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(nrf_cloud_mqtt_multi_service)
 zephyr_compile_definitions(PROJECT_NAME=${PROJECT_NAME})
 
+# Include certs files if enabled.
+# PLEASE NOTE: This is for testing purposes only! CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES stores
```

Comment:
seems like this is sufficient here:
```
# Include certs files if enabled.
zephyr_include_directories_ifdef(CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES certs)
```
the other comments have been stated elsewhere (especially if added to kconfig, see other comment)

Reply:
I'm worried someone is going to stumble into this and not check other docs -- I'll definitely add a warning in that location, but I'd like to keep the warning here as well

Reply:
But there is nothing for the user to do here, even if they stumble into this file...
they will need to seek out the kconfig used in this conditional include.

Reply:
So you're saying remove the instructions and the warning?

I can remove the instructions, but I'd rather keep the warning. I'd rather take every opportunity (or at least, every ***practical*** opportunity) to warn the user

Reply:
> So you're saying remove the instructions and the warning?

yes, the instructions and warnings are found elsewhere.
this is not a practical place imo.

Reply:
Alright, if you think its better to not warn the user here, I'll remove it